funnel/views/schedule.py

```diff
@@ -249,9 +288,22 @@ def edit_schedule(self):
         return {
             'project': self.obj,
             'proposals': proposals,
-            'from_date': date_js(self.obj.schedule_start_at),
-            'to_date': date_js(self.obj.schedule_end_at),
-            'timezone': self.obj.timezone.utcoffset(datetime.now()).total_seconds(),
+            'from_date': (
+                localize_timezone(
+                    self.obj.schedule_start_at, tz=self.obj.timezone
+                ).isoformat()
+                if self.obj.schedule_start_at
+                else None
+            ),
+            'to_date': (
+                localize_timezone(
+                    self.obj.schedule_end_at, tz=self.obj.timezone
+                ).isoformat()
+                if self.obj.schedule_start_at
+                else None
+            ),
+            'timezone_offset': self.obj.timezone.utcoffset(datetime.now()).total_seconds(),
```

Comment:
As above.

Reply:
Since it is being repeated, it should perhaps be a property on the Project model as `timezone_offset`.

Reply:
Frontend is using timezone to calculate the date & time so removing the timezone offset